src/main/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/resource/OffenceController.kt

```diff
@@ -33,6 +36,21 @@ class OffenceController(
     return offenceService.findOffencesByCode(offenceCode)
   }
 
+  @GetMapping(value = ["/sdrs/code/{offenceCode}"])
```

Comment:
This endpoint is only used to test the interface (hence is temporary). Once the integration is confirmed in the dev environment will start the 'real' work